iocore/net/P_QUICNet.h

```diff
@@ -64,12 +59,10 @@ struct QUICPollCont : public Continuation {
   // Atomic Queue to save incoming packets
   ASLL(QUICPollEvent, alink) inQueue;
 
+private:
```

Comment:
Please prepend "_" to private members' names

Reply:
get it .

Reply:
fixed !